src/classes/issues-processor.ts

```diff
@@ -268,6 +315,109 @@ export class IssuesProcessor {
       );
     }
 
+    if (this.options.mlflow) {
```

Comment:
This if block has some return statements to prevent the bot from staling issues that should not be staled.